### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by glaucocustodio**
Add missing comma.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ passed falls within the ```except``` list or when it falls entirely outside the
 
 ```ruby
 params do
-  requires :number, type: Integer, values: { value: 1..20 except: [4,13], except_message: 'includes unsafe numbers', message: 'is outside the range of numbers allowed' }
+  requires :number, type: Integer, values: { value: 1..20, except: [4,13], except_message: 'includes unsafe numbers', message: 'is outside the range of numbers allowed' }
```
Comment (namusyaka) [inline README.md]: @glaucocustodio Thanks! I want you to fix another a whitespace. `except: [4, 13]`

**Comment by namusyaka** [comment]
@glaucocustodio If you only change documentation, you can skip ci by adding `[ci skip]` to your commit message ;)

File: README.md (hunk 1 above)
Comment (glaucocustodio) [inline README.md]: Done ;)

**Comment by namusyaka** [comment]
Sorry to bother you again. Could you rebase and squash your commits?

**Comment by glaucocustodio** [comment]
Done ;). Thank you.

**Comment by namusyaka** [comment]
@glaucocustodio Perfect. Thank you so much!

**Pull request merged**